models/articles.models.js

```diff
@@ -15,6 +16,19 @@ function fetchArticles() {
     })
 }
 
-module.exports = {fetchArticleById, fetchArticles};
+function updateArticleById({article_id}, {inc_votes}) {
+    if(isNaN(inc_votes)) return Promise.reject({status: 400, msg: 'inc_votes must be a number'});
```

Comment:
this logic is unnecessary as PSQL will throw an error for you if the value isn't the correct data type